_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
```

Comment:
(a) It's hard to constrain what an API user need so it's a bit of defensive programming...who knows, maybe you want the mother's maiden name of the user and the user's spouse?

(b) if there is a collision, we enter undefined behavior in the browser which is hard to track down.

Since we're in a component world, we should assume that we cannot control what the users will do with composing our components.